tests/test_integrations/test_email/test_application_context.py

```diff
@@ -37,6 +39,11 @@
     SEASONAL_RESERVATION_CHECK_BOOKING_DETAILS_LINK_SV,
 )
 
+# Applied to all tests
+pytestmark = [
+    pytest.mark.django_db,
+]
+
```

Comment:
I wouldn't add this on the top level in this case, since most of the tests in this file don't need database access. It just makes them run slower.